tests/DocBlock/AnnotationTest.php

```diff
@@ -161,7 +164,7 @@ public function testGetTag(int $index, string $tag): void
     }
 
     /**
-     * @return iterable<int, array{int, string}>
+     * @return iterable<int, _PhpTokenArray>
```

Comment:
not a token array

Reply:
@gharlan , feel free to simply send a PR, if possible.